src/contracts/core/RewardsCoordinator.sol

```diff
@@ -340,6 +349,83 @@ contract RewardsCoordinator is
         }
     }
 
+    /**
+     * @notice Validate a PerformanceRewardsSubmission. Called from `createAVSPerformanceRewardsSubmission`.
+     * @param performanceRewardsSubmission PerformanceRewardsSubmission to validate.
+     * @return total amount to be transferred from the avs to the contract.
+     */
+    function _validatePerformanceRewardsSubmission(
+        PerformanceRewardsSubmission calldata performanceRewardsSubmission
+    ) internal view returns (uint256) {
+        require(
+            performanceRewardsSubmission.strategiesAndMultipliers.length > 0,
+            "RewardsCoordinator._validatePerformanceRewardsSubmission: no strategies set"
+        );
+        require(
+            performanceRewardsSubmission.operatorRewards.length > 0,
+            "RewardsCoordinator._validatePerformanceRewardsSubmission: no operators rewarded"
+        );
+
+        uint256 totalAmount = 0;
+        address currOperatorAddress = address(0);
+        for (uint256 i = 0; i < performanceRewardsSubmission.operatorRewards.length; ++i) {
+            OperatorReward calldata operatorReward = performanceRewardsSubmission.operatorRewards[i];
+            require(
+                operatorReward.operator != address(0),
+                "RewardsCoordinator._validatePerformanceRewardsSubmission: operator cannot be 0 address"
+            );
+            require(
+                currOperatorAddress < operatorReward.operator,
+                "RewardsCoordinator._validatePerformanceRewardsSubmission: operators must be in ascending order to handle duplicates"
+            );
+            currOperatorAddress = operatorReward.operator;
+            require(
+                operatorReward.amount > 0,
+                "RewardsCoordinator._validatePerformanceRewardsSubmission: operator reward amount cannot be 0"
+            );
```

Comment:
What are the gas costs for 200,500,1000 operators?

Reply:
Is that question wrt to this check or in general how much it would cost if 200,500,1000 operators were passed in calldata?

Reply:
The latter

Reply:
Here's a spreadsheet based on 64 bytes/operator calldata: https://docs.google.com/spreadsheets/d/1MOFK9wJQwmcktVywyiXOqyLYPNfMNo0ko9QU0bA2ua4/edit?gid=0#gid=0

Reply:
Do you want me to document that somewhere? Or some other form of gas report using test cases?

Reply:
Sounds good. I've noted it in the Exit criteria for technical docs.